src/containers/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -442,9 +442,11 @@ const ProfilePage = ({
     ]
 
     if (
-      profileHasCollectiblesTierRequirement &&
-      (profileHasVisibleImageOrVideoCollectibles ||
-        (profileHasCollectibles && isUserOnTheirProfile))
+      // `has_collectibles` is a shortcut that is only true iff the user has a modified collectibles state
+      profile?.has_collectibles ||
```

Comment:
Not sure if this is correct. This seems like the collectibles tab would be displayed in the following scenarios even though it shouldn't:
- profile used to have a tier silver or higher, but not has downgraded tiers
OR
- profile has collectibles but all of them are hidden and user is not on their profile